.github/pull-request-template.md

```diff
@@ -1,19 +1,68 @@
-<!--   Thank you for your contribution. Before you submit the pull request:
-1. Follow contributing guidelines, templates, the recommended Git workflow, and any related documentation.
-2. Read and submit the required Contributor Licence Agreements (https://github.com/kyma-project/community/blob/main/CONTRIBUTING.md#agreements-and-licenses).
+<!--   
+
+Thank you for your contribution!
+
+Before submitting your pull request, please follow these steps:
+
+1. Adhere to contributing guidelines, templates, the recommended Git workflow, and related documentation.
+2. Read and submit the required Contributor Licence Agreements (https://github.com/kyma-project/community/blob/main/docs/contributing/02-contributing.md).
 3. Test your changes and attach their results to the pull request.
 4. Update the relevant documentation.
+5. Fill in the checklist below.
 
-If the pull request requires a decision, follow the [decision-making process](https://github.com/kyma-project/community/blob/main/governance.md) and replace the PR template with the [decision record template](https://github.com/kyma-project/community/blob/main/.github/ISSUE_TEMPLATE/decision-record.md).
+If the pull request requires a decision, follow the [decision-making process](https://github.com/kyma-project/community/blob/main/docs/governance/01-governance.md) and replace the PR template with the [decision record template](https://github.com/kyma-project/community/blob/main/.github/ISSUE_TEMPLATE/decision-record.md).
 -->
 
-**Description**
+## Description
+
+<!-- Add a brief description of WHAT was done and WHY. -->
 
 Changes proposed in this pull request:
 
-- ...
-- ...
 - ...
 
-**Related issue(s)**
+## Related Issues and Documents
+
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+Closes:
+
+Related issues:
+
+## Traceability
+
+- [ ] The PR is linked to a GitHub Issue.
+- [ ] New features have a milestone label set.
+- [ ] New features have defined acceptance criteria in a corresponding GitHub Issue, and all criteria are satisfied with this PR.
+- [ ] The corresponding GitHub Issue has a respective `area` label.
+
+## Testability
+
+The feature is unit-tested:
+
+- [ ] Yes.
+- [ ] No, because unit tests are not needed.
+- [ ] No, because of ...
+
+The feature is e2e-tested:
+
+- [ ] Yes.
+- [ ] No, because e2e-tests are not needed.
+- [ ] No, because of ...
+
+<!--
+Please describe the tests you ran to verify your changes if needed. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+-->
+Tests conducted for the PR:
+
+- Test description goes here
+
+## Checklist
+
+- [ ] My code follows the style guidelines of this project.
```

Comment:
Can we link to those style guidelines?

Reply:
Unfortunately, we have them as "tribal knowledge" for now, so there is no document to be linked. This item in a checklist is an instigation to formalise them.